### ISSUE TITLE
High Power usage causing serious lag

### ISSUE THREAD

**Issue by Zorotic** [mumble-voip/mumble#6054]
### The issue

On windows 10 mumble power usage will bolt up to Very High and cause the program to lag seriously. if I click and hover over mumble in task manager the power usage will go back to where it should be.. this is the weirdest issue I've seen and I have no idea why its happening (please help)

### Mumble version

1.5.517

### Mumble component

Client

### OS

Windows

### Additional information

PC specs shouldn't be an issue mumble is the only program that does this. 

CPU: Ryzen 9 3900x
RAM: 32gb
PSU is 750w 

https://i.imgur.com/zZQioBy.png

**Comment by Krzmbrzl** [comment]
Hm.... a couple of questions for you:
1. Do you have some sort of joystick or other analogue input device plugged into your computer?
2. Did you have an earlier version of Mumble (pre 1.5 - preferably one of 1.4.x) installed on your system before and if so did that also show this behavior?

**Comment by Zorotic** [comment]
Yes actually I do, I have a fanatec handbrake plugged in, and yes it does this on any version I install (I removed the old version before updating)

First I had 1.4 from the website and decided to try the rc version from GitHub. What are my options here?

Thanks!

**Comment by Krzmbrzl** [comment]
Could you try and are whether unplugging all analogue input devices (joysticks and the like) and then check if the issue persists? These devices tend to continuously send input events to the computer that Mumble has to process.

**Comment by Zorotic** [comment]
Unplugged my handbrake and it fixed the issue. Only issue with that is I'm going to be utilizing the mumble link plugin for proximity chat in Assetto Corsa on a server I have setup. is there a way to make mumble ignore the handbrakes input?

**Comment by Krzmbrzl** [comment]
@davidebeatrici this is yet another issue with that ridiculous shortcut event flooding...

> is there a way to make mumble ignore the handbrakes input?

Internally, the events are already ignored (afaik) but the API simply spams these events so we have to spent valuable processing time to sort through them...

**Comment by Zorotic** [comment]
Gotcha, thanks for the explanation!

**Comment by Hartmnt** [comment]
> Gotcha, thanks for the explanation!

Hey, could you try disabling "global shortcuts" in the Mumble settings to see if this fixes the lags?

**Comment by Krzmbrzl** [comment]
Closing this in favor of https://github.com/mumble-voip/mumble/issues/6696
